src/content/docs/en/recipes.mdx

```diff
@@ -1,7 +1,20 @@
 ---
-title: old recipes page
-description: old recipes page
+title: Official Astro recipes
```

Comment:
Looks good! But I wonder if we should reword the title of the page:
* to avoid `<h1>Official Astro recipes</h1>` then `<h2>Official recipes</h2>` (this is the point that bothers me the most... but I can deal with it 😄 )
* to make the `Community resources` section fit better on the page (since they are not official)
* to reflect the name in the sidebar (even if it doesn't necessarily have to be word for word...)
```suggestion
title: How-to recipes
```